src/components/AllBreweriesHome/AllBreweriesHome.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from 'react';
+import BreweryInterface from "../BreweryCard/BreweryCard";
+
+export default function AllBrieweriesHome() {
+    const [breweries, setBreweries] = useState([]);
+    useEffect(() => {
+        fetch('https://api.openbrewerydb.org/v1/breweries?fbclid=IwZXh0bgNhZW0CMTAAAR1QKdPcqOwBI51QmYcnitDSFfKOnHlOLDSbxXadLJZgJJWfQ797QPTltHA_aem_ZmFrZWR1bW15MTZieXRlcw')
+            .then((res) => {
+                return res.json();
+            })
+            .then((data) => {
+                console.log(data);
+                setBreweries(data);
+            });
+    }, []);
+
+    return (
+        <div className="allBrieweriesHome">
```

Comment:
css classes should follow the kebab case naming convension - https://smartwp.com/kebab-case/ or the BEM convension - https://css-tricks.com/bem-101/

Reply:
i chose the kebab case and changed all class names